File: Problems and Solutions/0-fizzbuzz.java
// Java
import java.util.Scanner;

class FizzBuzz
{
  public void solve() {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    sc.close();

    for (int i = 1; i <= n; i++) {
      if (i % 3 == 0 && i % 5 == 0)
        System.out.println("FizzBuzz");
      else if (i % 3 == 0)
        System.out.println("Fizz");
      else if (i % 5 == 0)
        System.out.println("Buzz");
      else
        System.out.println(i);
    }
  }

  public static void main(String[] args)
  {
    new FizzBuzz().solve();
  }
}

File: Problems and Solutions/0-fizzbuzz.cpp
// C++
#include <iostream>
using namespace std;

class FizzBuzz
{
  public:
    void solve() {
      int n; cin >> n;
      for (int i = 1; i <= n; i++) {
        if (i % 3 == 0 && i % 5 == 0)
          cout << "FizzBuzz\n";
        else if (i % 3 == 0)
          cout << "Fizz\n";
        else if (i % 5 == 0)
          cout << "Buzz\n";
        else
          cout << i << "\n";
      }
    }
};

int main() {
  FizzBuzz fb;
  fb.solve();

  return 0;
}
